web/src/App.jsx

```diff
@@ -28,9 +28,9 @@ import { useInstallerL10n } from "./context/installerL10n";
 import { useInstallerClientStatus } from "~/context/installer";
 import { useProduct, useProductChanges } from "./queries/software";
 import { CONFIG, INSTALL, STARTUP } from "~/client/phase";
-import { BUSY } from "~/client/status";
```

Comment:
As discussed offline, please check if _client/status_ still needed.

Reply:
It is still needed by the storage part. But `client/phase` is not, so I have removed it.